EIPS/eip-721.md

```diff
@@ -21,7 +21,7 @@ The following standard allows for the implementation of a standard API for NFTs
 We considered use cases of NFTs being owned and transacted by individuals as well as consignment to third party brokers/wallets/auctioneers ("operators"). NFTs can represent ownership over digital or physical assets. We considered a diverse universe of assets, and we know you will dream up many more:
 
 - Physical property — houses, unique artwork
-- Virtual collectables — unique pictures of kittens, collectable cards
+- Virtual collectibles — unique pictures of kittens, collectible cards
```

Comment:
Collectible is a noun. Collect*a*ble is an adjective.
```suggestion
- Virtual collectibles — unique pictures of kittens, collectable cards
```

Reply:
Thanks for the feedback! I'm not opposed to "collectable cards" if there's consensus that it's clearer.

The line https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md?plain=1#L361 was already using "collectible tokens" and I wanted to match that.

Also, https://www.merriam-webster.com/dictionary/collectible is defined as both an adjective and a noun. And also https://en.wikipedia.org/wiki/Trading_card which uses "collectible card" in the same sense.

I'm more than happy to amend the PR.

Reply:
Actually, OED doesn't even recognize "collectible" as a word. I think that the best thing to do is leave this be, since it's a final EIP.